pkg/admin/services/provider_service_impl_test.go

```diff
@@ -0,0 +1,224 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package services
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/apache/dubbo-admin/pkg/admin/model"
+)
+
+func TestProviderServiceImpl_FindServices(t *testing.T) {
```

Comment:
It would be better if you can initialize `cache.InterfaceRegistryCache` with some specially designed values for testing ProviderServiceImpl.

Reply:
I can do it, but I worry about that if someone run the test with different values of InterfaceRegistryCache, the test will fail. If it is nothing, I will do it.